.travis.yml

```diff
@@ -30,17 +30,15 @@ matrix:
 
     # Without ENVIRONMENT variables
     - php: 7.3
+      install:
+        - composer create-project sulu/sulu-minimal ../test-create-project-dir --repository="{\"type\":\"path\",\"url\":\"./\"}" --stability=dev -n
```

Comment:
This change was needed else it did not fail on install:
See: https://travis-ci.org/sulu/sulu-minimal/jobs/521738947